docs/source/distributions.rst

```diff
@@ -404,6 +404,22 @@ ZeroSumNormal
     :show-inheritance:
     :member-order: bysource
 
+DoublyTruncatedPowerLaw
+^^^^^^^^^^^^^^^^^^^^^^^
+.. autoclass:: numpyro.distributions.continuous.DoublyTruncatedPowerLaw
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
+
+LowerTruncatedPowerLaw
+^^^^^^^^^^^^^^^^^^^^^^
+.. autoclass:: numpyro.distributions.continuous.LowerTruncatedPowerLaw
+    :members:
+    :undoc-members:
+    :show-inheritance:
+    :member-order: bysource
```

Comment:
Could you add those doc entries to the right alphabet order?

Reply:
Do you want me to reorder the distribution position by their alphabetical order or do you want `:member-order:` to be alphabetical?

Reply:
Actually, I think we should move those distributions to the Truncated section https://num.pyro.ai/en/stable/distributions.html#truncated-distributions and move the implementation to the `distributions/truncated.py` script

Reply:
> reorder the distribution position by their alphabetical order

I meant this one

Reply:
> Actually, I think we should move those distributions to the Truncated section https://num.pyro.ai/en/stable/distributions.html#truncated-distributions and move the implementation to the `distributions/truncated.py` script

I was doing this in the first place. The reason for not doing so is that I observed all the distributions in that file are truncated versions of different "untruncated" distributions. `DoublyTruncatedPowerLaw` and `LowerTruncatedPowerLaw` are by definition a truncated distribution just like `HalfNormal`. If still like to see them in that file I can move it.

Also at the time of writing three CI actions are running please stop the older two they are wasting energy and resources.

Reply:
It is better to move to `truncated.py`. The api there is more aligned with the distributions in this PR.

> at the time of writing three CI actions are running please stop the older two

Thanks! For some reasons, the tests for those distributions are very slow. I guess there are some infinite loops are going on? (maybe due to the definition of custom_jvp)

Reply:
> I guess there are some infinite loops are going on? (maybe due to the custom jvp rules)

custom jvp rules are implemented by @InfinityMod he can share his insights.

> It is better to move to `truncated.py`. The api there is more aligned with the distributions in this PR.

I will shift them!